### PR TITLE
scaffolding for HCU transaction to be executed on Monday, July 10th

### PR DESCRIPTION
**Pull request by vishalchangrani**

### CODE REVIEW

File: templates/set_version_boundary.cdc [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import NodeVersionBeacon from 0xNODEVERSIONBEACONADDRESS
```
Comment (vishalchangrani) [inline templates/set_version_boundary.cdc:1]: @janezpodhostnik what should this address be?

File: transactions/set-version-boundary/2023/jul-10/arguments.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "type": "UInt8",
```
Comment (vishalchangrani) [inline transactions/set-version-boundary/2023/jul-10/arguments.json:3]: what should these be for this HCU?

File: templates/set_version_boundary.cdc (hunk 1 above)
Comment (janezpodhostnik) [reply templates/set_version_boundary.cdc:1]:
```suggestion
import NodeVersionBeacon from 0xe467b9dd11fa00df
```

mainnet service account

File: transactions/set-version-boundary/2023/jul-10/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+> July 10th, 2023
+
+
+This transaction sets the height at which the height Coordinated upgrade occurs on mainnet.
+
+## New method using Flow CLI multi-sign
+
+1. Flow generates the Signature Request ID on the [site](https://flow-multisig-git-service-account-onflow.vercel.app/mainnet) for the `set_version_boundary.cdc` transaction with the given args.
+
+2. Signers sign the staking account key using the web tool.
```
Comment (janezpodhostnik) [inline transactions/set-version-boundary/2023/jul-10/README.md:11]:
```suggestion
2. Signers sign the service account key using the web tool.
```

File: transactions/set-version-boundary/2023/jul-10/arguments.json (hunk 2 above)
Comment (janezpodhostnik) [reply transactions/set-version-boundary/2023/jul-10/arguments.json:3]: see comment below.

**Comment by vishalchangrani** [comment]
Thank you @janezpodhostnik !

**Pull request merged**